plone/app/layout/viewlets/social.py

```diff
@@ -96,7 +102,7 @@ def _get_tags(self):
                 scales = self.context.restrictedTraverse('@@images', None)
                 if scales:
                     try:
-                        image = scales.scale('image', scale='great')
+                        image = scales.scale('image', scale=SOCIAL_IMAGE_SCALE)
```

Comment:
I am not a fan customization using module variables.

Can we have this as a class attribute?
This will allow (for example) to customize the behavior per site in case of multiple sites served by the same zope instance.

Reply:
@ale-rt yes indeed.

This was one of the option in my comment https://github.com/plone/plone.app.layout/issues/323#issuecomment-1242067612.
OK for me too, we can modify the PR 👍